artifactory/commands/buildinfo/publish.go

```
@@ -91,6 +93,35 @@ func (bpc *BuildPublishCommand) Run() error {
 	if err != nil {
 		return err
 	}
+
+	buildTime, err := time.Parse(buildinfo.TimeFormat, buildInfo.Started)
+	if errorutils.CheckError(err) != nil {
+		return err
+	}
+
+	// Convert from nanoseconds to milliseconds
+	timestamp := buildTime.UnixNano() / 1000000
+	artVersioin, _ := servicesManager.GetVersion()
```

Let's not ignore the err returned from the ```GetVersion()``` method.